src/SME.SGP.Aplicacao/CasosDeUso/ConselhoClasse/ObterNotasFrequenciaUseCase.cs

```diff
@@ -200,11 +201,22 @@ public async Task<ConselhoClasseAlunoNotasConceitosRetornoDto> Executar(Conselho
             var ordenacaoGrupoArea = (await mediator.Send(new ObterOrdenacaoAreasConhecimentoQuery(disciplinasDaTurma, areasDoConhecimento))).ToList();
             var retorno = new ConselhoClasseAlunoNotasConceitosRetornoDto();
             var gruposMatrizesNotas = new List<ConselhoClasseAlunoNotasConceitosDto>();
+            
+            var componenteRegenciaTurmaPai = new DisciplinaResposta();
+            if (disciplinasDaTurmaEol.Any(x=>x.Regencia == true))
+            {
+                componenteRegenciaTurmaPai = (await mediator.Send(new ObterComponentesCurricularesPorTurmaCodigoQuery(turma.CodigoTurma))).FirstOrDefault(x => x.Regencia == true);
+                disciplinasDaTurmaEol.Add(new DisciplinaDto() { CodigoComponenteCurricular = componenteRegenciaTurmaPai.CodigoComponenteCurricular, Regencia = componenteRegenciaTurmaPai.Regencia });
```

Comment:
aqui pode ter a chance desse componente ser nulo e na hora que for obter o "componenteRegenciaTurmaPai.CodigoComponenteCurricular" dar erro de referência, verificar um tratamento para esse caso.